app/views/questions/show.haml

```diff
@@ -1,4 +1,4 @@
 %h1
   = @wizard.name
 
-= react_component("AgedCareWidget", props: @props)
+= react_component("AgedCareWidget", props: @widget_properties)
```

Comment:
Feels like there is some duplication between controller & view. Rather than explicitly setting a widget properties variable, would it be possible to simply: 

`react_component("AgedCareWidget", props: render(template: "/questions/show.json.jbuilder"))`

.... ?